components/User.tsx

```diff
@@ -1,12 +1,11 @@
-import React, { Component } from 'react'
-import { View, Text, Image, StyleSheet, TextInput, ScrollView } from "react-native"
+import React from 'react'
+import { View, Text, Image, StyleSheet, TouchableOpacity } from "react-native"
 import Badges from "./Badges"
 import { connect } from "react-redux"
 
 const User = props => {
-    console.log(props.user.profile_photo, "this is a photo")
 	return (
-		<View style={styles.user}>
+		<TouchableOpacity style={styles.user} onPress={() => props.showProfile(props.user)}>
```

Comment:
Changed "View" to "TouchableOpacity" to provide feedback to the user, when the element is clicked. Adding onPress method to show the profile of the clicked user